0666-return-to-hnt.md

```diff
@@ -0,0 +1,155 @@
+# HIP 666: Return to HNT
```

Comment:
```suggestion
# HIP XXXX: Return to HNT
```

Reply:
Standard @ferebee. Needs 😈 on things.

Reply:
Seems like veMOBILE and veIOT holders should also get to vote.

Reply:
@incrementis7 probably worth discussing in channels but technically the tokens were created by HNT voters (prior to Solana migration) and the treasuries are managed by these participants. HIPs 51 and 77 are probably the most relevant.